tools/deploy_assisted_installer_configmap.py

```diff
@@ -51,6 +50,7 @@ def handle_arguments():
 RELEASE_SOURCES = os.environ.get("RELEASE_SOURCES", "")
 OPENSHIFT_RELEASE_SYNCER_INTERVAL = os.environ.get("OPENSHIFT_RELEASE_SYNCER_INTERVAL", "30m")
 IGNORED_OPENSHIFT_VERSIONS = os.environ.get("IGNORED_OPENSHIFT_VERSIONS", "")
+JWKS_URL = os.environ.get("JWKS_URL", "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs")
```

Comment:
let’s keep the command line, better to have  backward compatibility 


```suggestion
parser.add_argument("--jwks-url", default=os.environ.get('JWKS_URL', "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs")
```